src/journal/journald-syslog.c

```diff
@@ -386,8 +391,18 @@ void server_process_syslog_message(
         }
 
         message = strjoina("MESSAGE=", msg);
-        if (message)
-                iovec[n++] = IOVEC_MAKE_STRING(message);
+        iovec[n++] = IOVEC_MAKE_STRING(message);
+
+        if (store_original) {
+                const size_t hlen = strlen("_ORIGINAL_MESSAGE=");
```

Comment:
strlen() → STRLEN()

Reply:
STRLEN is only useful to avoid VLAs, I don't think we need to use it otherwise.